nlptest/utils/SoundsLikeFunctions.py

```diff
@@ -0,0 +1,76 @@
+import re
+from ..transform.utils import CMU_dict
+
+
+class G2p(object):
+    def __init__(self):
+        super().__init__()
+        self.cmu = CMU_dict
+        
+    def __call__(self, text):
+        prons = []
+        text = text.lower()
+        text = re.sub("[^ a-z'.,?!\-]", "", text)
+
+        if text in self.cmu:
+            return self.cmu[text][0]
+        elif text in self.CMU_dict:
+            return self.CMU_dict[text]
+        else:
+
+            return prons
+g2p = G2p()
+        
+class Word_Functions():
```

Comment:
remove brackets after the class name.

**Word_Functions()**: -> **Word_Functions**:

do the same thing for classes except those are inherited from other types.